app/controllers/api/teams.php

```diff
@@ -73,6 +75,7 @@
                 'joined' => \time(),
                 'confirm' => true,
                 'secret' => '',
+                'search' => implode(' ', [$membershipId, $user->getId()])
```

Comment:
Do we want to add the user name and email here?

Reply:
thought about it first, but we would need to maintain it when a name or email is updated in both account and user endpoints.

Reply:
Got it, let's leave it for now